### ISSUE TITLE
Getting started (documentation) is imposible to follow

### ISSUE THREAD

**Issue by vchekan** [grafana/grafana#1448]
Getting Started chapter in documentation http://grafana.org/docs/features/intro/ shows how to create a new graph, but it is using animated gif which can not be paused. Also it is not possible to understand what is being clicked because it happens so quick. 
Suggestion: redo it as video, with voice comments, and perhaps with capturing software which shows clicks in the video, and replace animated gifs with links to youtube.

**Comment by torkelo** [comment]
good feedback, will be reviewing and updating documentation for Grafana 2.0 release. 

Was there anything specific in the getting started guide, in the annotated screenshots, and the animated gifs that you felt lacking (beside to fast, and hard to see what is clicked). Was there anything specific in the getting started experience (the UI, etc) that should deserve more focus or that was particularly unintuitive?

**Comment by vchekan** [comment]
Well, after I watched it many times, it became clear what is going on, but it took level of effort which is not reasonable for reading documentation. For example, at the beginning, when mouse is hovering over some elements, there are some hints. They disappear so quickly that I do not have time even to think either it was an intention to show me something important or unintended. 

Consider for example this article:
http://www.rittmanmead.com/2015/02/obiee-monitoring-and-diagnostics-with-influxdb-and-grafana/
Scroll to "Building Grafana Dashboards on Metrics Held in InfluxDB" section. They just took snapshots with circled steps. It is way easier to follow and takes less time to produce.

Overall, I would structure the documentation around 2 sections: "happy path": something what will be set up by 90% of users. And "specific" chapters which would highlight metrics acquisition on windows, linux, java, high load advices, how to use function X to troubleshoot and monitor Y. 

Right now documentation is funtion-centric. Plus there is no from-simple-to-advanced flow. For example, Search feature is on the left panel, top level. But newcomer would be the least interested in this feature because it becomes useful only when you created a lot of graphs. Which means advanced user. Meanwhile super useful for novice features, such as aggregating metrics, or comparing metrics to itself week ago are not easy to spot. While I visited the grafana documentation site the first, I did not realize how incredibly useful those 2 functions are. I got this realization when I was browsing some random grafana presentations around the web. Those functions might be somwhere in the doc site, I just haven't found them yet.

I do realize that documentation should address multiple audiences.
Newby needs quick and visual capture of killing features (and grafana does have them). And once hooked up, simple way to start simple scenario.
Experianced users need "what's new" and reference-type documentation.

All the above should not tax developers too hard, because nobody likes writing documentation :) And even more important is the question of minimization cost of documentation update. Perhaps screenshot-type guide instead of animated gif is a good one. Better clarity while less developers time. Another one, is grafana-2.0 features presentation site http://play.grafana.org/v2/login which is great for purpose of capturing attention because in several demo pictures it gives very good idea of practical use cases which are not apparent from reading list of functions in documentation.

That's my random thoughts on the subject.
